src/main/java/net/mcreator/element/converter/fv32/ItemDispenserToItemExtensionConverter.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * MCreator (https://mcreator.net/)
+ * Copyright (C) 2012-2020, Pylo
+ * Copyright (C) 2020-2022, Pylo, opensource contributors
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package net.mcreator.element.converter.fv32;
+
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import net.mcreator.element.GeneratableElement;
+import net.mcreator.element.ModElementType;
+import net.mcreator.element.converter.IConverter;
+import net.mcreator.element.parts.MItemBlock;
+import net.mcreator.element.parts.Procedure;
+import net.mcreator.element.types.ItemExtension;
+import net.mcreator.workspace.Workspace;
+import net.mcreator.workspace.elements.FolderElement;
+import net.mcreator.workspace.elements.ModElement;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class ItemDispenserToItemExtensionConverter implements IConverter {

```

Comment:
What is _item dispenser_? I'd name this class `ItemDispenseBehaviorToItemExtensionConverter`.